### PR TITLE
Add error codes

### PR DESCRIPTION
**Pull request by christyjacob4**
Screenshot 2022-02-07 at 3 15 18 AM

### CODE REVIEW

File: app/views/docs/error-codes.phtml [hunk 1]
```diff
@@ -106,4 +116,27 @@
     "code": 400
 }
 </code></pre>
-</div>
\ No newline at end of file
+</div>
+
+<h2><a href="/docs/functions#environmentVariables" id="environmentVariables">Error Codes</a></h2>
```
Comment (eldadfux) [inline app/views/docs/error-codes.phtml:121]:
```suggestion
<h2><a href="/docs/functions#environmentVariables" id="environmentVariables">Error Codes</a></h2>
```

The link and ID are wrong.

File: app/views/docs/error-codes.phtml [hunk 2]
```diff
@@ -106,4 +116,27 @@
     "code": 400
```
Comment (eldadfux) [inline app/views/docs/error-codes.phtml:116]: We should update the code example.

**Review by eldadfux — CHANGES_REQUESTED** [review]
What should we do with the 1st table in the page? I feel the difference between them is not very clear for the reader.

File: app/views/docs/error-codes.phtml (hunk 2 above)
Comment (christyjacob4) [reply app/views/docs/error-codes.phtml:117]: done 👍

File: app/views/docs/error-codes.phtml (hunk 1 above)
Comment (christyjacob4) [reply app/views/docs/error-codes.phtml]: fixed

**Comment by christyjacob4** [comment]
@eldadfux  the first table talks about the convention we follow when deciding an HTTP status code for our errors. And the second table talks more about Appwrite specific error codes.

**Review by eldadfux — APPROVED** [review]

**Comment by eldadfux** [comment]
I would only consider adding the version we started supporting each error code. We have something similar for env vars.

**Comment by christyjacob4** [comment]
@eldadfux In this case, I'm not sure we can follow the convention we use for ENV Vars.
Screenshot 2022-02-08 at 7 41 46 PM
 
Since we introduced all these error codes in 0.13. literally every row in the table will have this information and it'll just make the table very noisy. 

It's probably a better option to mention that this feature was introduced in `0.13` once in one of the paragraphs above. 

From 0.14 onwards, we can start following the ENV Vars convention

**Comment by christyjacob4** [comment]
Screenshot 2022-02-08 at 8 37 24 PM


@eldadfux  updated the description to add the note

**Review by gewenyu99 — APPROVED** [review]
LGTM <3

**Comment by christyjacob4** [comment]
## Latest Preview

Screenshot 2022-02-09 at 2 49 56 AM

**Review by Meldiron — APPROVED** [review]
LGTM

**Comment by eldadfux** [comment]
Changed target branch to 0.13

**Pull request merged**